<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtQuery</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QsciScintilla" name="textEditQuery">
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>Query</string>
    </property>
    <addaction name="actionExecute"/>
    <addaction name="actionExplainAnalyze"/>
    <addaction name="actionExplain"/>
    <addaction name="actionFormat"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionDatabaseLayout"/>
    <addaction name="actionDataOutput"/>
    <addaction name="actionMessages"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionComment"/>
    <addaction name="actionUncomment"/>
    <addaction name="separator"/>
    <addaction name="actionIndent"/>
    <addaction name="actionUnindent"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuQuery"/>
   <addaction name="menuWindows"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetDataOutput">
   <property name="windowTitle">
    <string>Data Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTableView" name="tableViewDataOutput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExecute"/>
   <addaction name="actionExplainAnalyze"/>
   <addaction name="actionExplain"/>
   <addaction name="actionFormat"/>
   <addaction name="actionDocumentation"/>
   <addaction name="actionMessages"/>
   <addaction name="actionDataOutput"/>
   <addaction name="actionDatabaseLayout"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetMessages">
   <property name="windowTitle">
    <string>Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTextBrowser" name="messagesBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetDatabaseLayout">
   <property name="windowTitle">
    <string>Database Layout</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidgetDatabaseLayout">
       <column>
        <property name="text">
         <string notr="true">Database</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetDocumentation">
   <property name="windowTitle">
    <string>Documentation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0" colspan="7">
      <widget class="QWebView" name="webViewDocumentation">
       <property name="url">
        <url>
         <string>http://www.postgresql.org/docs/8.4/interactive/index.html</string>
        </url>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="LineEditDocumentationUrl"/>
     </item>
     <item row="2" column="0" colspan="7">
      <widget class="QProgressBar" name="progressBarDocumentation">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="toolButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/Stop.png</normaloff>:/resources/images/Stop.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButtonReload">
       <property name="text">
        <string>Reload</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/Refresh.png</normaloff>:/resources/images/Refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="toolButtonBackward">
       <property name="text">
        <string>Backward</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/Undo.png</normaloff>:/resources/images/Undo.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButtonForward">
       <property name="text">
        <string>Forward</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/Redo.png</normaloff>:/resources/images/Redo.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="toolButtonGo">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/Execute.png</normaloff>:/resources/images/Execute.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExecute">
   <property name="text">
    <string>Execute</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionExplainAnalyze">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Explain: Analyze</string>
   </property>
  </action>
  <action name="actionExplain">
   <property name="text">
    <string>Explain</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionFormat">
   <property name="text">
    <string>Format</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionDatabaseLayout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Database Layout</string>
   </property>
  </action>
  <action name="actionDataOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Data Output</string>
   </property>
  </action>
  <action name="actionMessages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUnindent">
   <property name="text">
    <string>Unindent</string>
   </property>
  </action>
  <action name="actionIndent">
   <property name="text">
    <string>Indent</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="text">
    <string>Comment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="actionUncomment">
   <property name="text">
    <string>Uncomment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+?</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>webViewDocumentation</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBarDocumentation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>918</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonStop</sender>
   <signal>pressed()</signal>
   <receiver>webViewDocumentation</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>918</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonReload</sender>
   <signal>pressed()</signal>
   <receiver>webViewDocumentation</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>918</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonForward</sender>
   <signal>pressed()</signal>
   <receiver>webViewDocumentation</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>918</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonBackward</sender>
   <signal>pressed()</signal>
   <receiver>webViewDocumentation</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>918</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMessages</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidgetMessages</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDatabaseLayout</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidgetDatabaseLayout</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDocumentation</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidgetDocumentation</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>925</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDataOutput</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidgetDataOutput</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetMessages</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionMessages</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetDataOutput</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionDataOutput</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetDocumentation</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionDocumentation</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidgetDatabaseLayout</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionDatabaseLayout</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
